src/Stratis.Features.FederatedPeg/Collateral/CollateralChecker.cs

```diff
@@ -152,34 +145,34 @@ private async Task<bool> UpdateCollateralInfoAsync(CancellationToken cancellatio
 
             if (addressesToCheck.Count == 0)
             {
+                this.logger.LogInformation("None of the federation members has a valid collateral amount to check.");
```

Comment:
0 is a valid amount. 
I'd change the message to `None of the federation members has collateral requirement configured`